github-notifications/main.py

```diff
@@ -0,0 +1,111 @@
+import os
+import functions_framework
+from slack import WebClient
+from slack.errors import SlackApiError
+
+import logging
+logging.basicConfig(level=logging.DEBUG)
+
+from dotenv import load_dotenv
+load_dotenv()
+
+from flask import Flask
+app = Flask(__name__)
+
+@functions_framework.http
+@app.route('/', methods=['POST'])
+def process_request(request):
+    webhook_payload = request.get_json()
+    webhook_payload_list = list(webhook_payload)  
```

Comment:
Looks like this is only referenced in the commented out code. Maybe a leftover we could get rid of?